assets/src/edit-story/components/form/dateTime/timePicker.js

```diff
@@ -0,0 +1,367 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable header/header */
+/* Disable reason: This file is based on Gutenberg Datepicker */
+
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import moment from 'moment';
+import styled, { css } from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+const TIMEZONELESS_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
+
+const DateTimeWrapper = styled.div`
+  margin-bottom: 1em;
+`;
+
+const Fieldset = styled.fieldset`
+  margin-top: 0.5em;
+  position: relative;
+`;
```

Comment:
Overall design is not correct here. Compare design:

![Screen Shot 2020-03-23 at 3 53 42 PM](https://user-images.githubusercontent.com/637548/77357495-a9985280-6d1e-11ea-943b-5fc6e60bcc38.png)

With actual component:

![Screen Shot 2020-03-23 at 3 53 34 PM](https://user-images.githubusercontent.com/637548/77357511-aef59d00-6d1e-11ea-80b0-726a1962762b.png)

Some discrepancies:

* Rounded corners on overall shape
* Padding in date-time picker
* Text inputs should be centered
* Input (30px), buttons (28px) and dropdown (28px) should be same height

And I now see that the "design" is just a screenshot - probably of the GB component. But it should then just be sharpened up a bit 😉

Reply:
This is currently how the original component looks like in my browser, quite similar to the one in this PR:
<img width="275" alt="Screenshot 2020-03-24 at 09 56 53" src="https://user-images.githubusercontent.com/3294597/77407022-cfbd0180-6db5-11ea-9d93-0ba3e92f0145.png">

I'll add some small fixes but we can leave the actual designing for when we have one, will keep it mostly as-is since it's just a screenshot in Figma files currently.

Reply:
Note that the screenshot is likely out of date as WordPress/GB has made tons of changes to its components in recent time.